README.md

```diff
@@ -86,6 +87,13 @@ Resources:
       <td>no</td>
       <td></td>
     </tr>
+    <tr>
```

Comment:
Could you add the two new parameters to the example at the beginning of the readme